monitoring-engine/src/main/java/io/jenkins/plugins/remotingopentelemetry/engine/DefaultEngineConfiguration.java

```diff
@@ -0,0 +1,47 @@
+package io.jenkins.plugins.remotingopentelemetry.engine;
+
+import io.jenkins.plugins.remotingopentelemetry.engine.config.ConfigOption;
+import io.jenkins.plugins.remotingopentelemetry.engine.config.Configuration;
+
+import javax.annotation.Nonnull;
+import java.util.UUID;
+import java.util.regex.Pattern;
+
+@Configuration
+public class DefaultEngineConfiguration implements EngineConfiguration {
+    @ConfigOption(env = "OTEL_EXPORTER_OTLP_ENDPOINT", required = true)
+    public String endpoint;
+
+    static private final int exporterTimeoutMillis = 30_000;
+
+    @ConfigOption(env = "SERVICE_INSTANCE_ID")
+    public String serviceInstanceId = UUID.randomUUID().toString();
+
+    @ConfigOption(env = "REMOTING_OTEL_METRIC_FILTER")
+    public Pattern metricsFilterPattern = Pattern.compile(".*");
```

Comment:
hey, you can define ".*" as a common constant

Reply:
It is possible, but overall it is an overkill when there is just one usage